lib/Dialect/FIRRTL/Transforms/EmitOMIR.cpp

```diff
@@ -548,7 +548,8 @@ void EmitOMIRPass::emitOptionalRTLPorts(DictionaryAttr node,
 
           // Emit the `width` field.
           buf.assign("OMBigInt:");
-          Twine(port.value().type.getBitWidthOrSentinel()).toVector(buf);
+          Twine::utohexstr(port.value().type.getBitWidthOrSentinel())
+              .toVector(buf);
```

Comment:
I was recently wondering if `toVector` appended or not :)

Reply:
It does :smile: